tests/serialization/test_dag_serialization.py

```diff
@@ -64,9 +64,9 @@
 from airflow.models.expandinput import EXPAND_INPUT_EMPTY
 from airflow.models.mappedoperator import MappedOperator
 from airflow.models.xcom import XCom
-from airflow.operators.empty import EmptyOperator
 from airflow.providers.cncf.kubernetes.pod_generator import PodGenerator
 from airflow.providers.standard.operators.bash import BashOperator
+from airflow.providers.standard.operators.empty import EmptyOperator
```

Comment:
why import it here if it is imported lazily in the file inside functions?

Reply:
Thats because it is in a type checking block used for mypy etc. But we import it normally in functions that need it for sake of usage

Reply:
if it's for mypy, shouldn't we use `if TYPE_CHECKING`?

Reply:
Right yeah, let me move it